src/components/editableSelect/index.tsx

```diff
@@ -1,163 +1,159 @@
-    if (!isOpen && search !== '' && (isMulti || search !== value[0])) {
-      handleToggle();
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [search]);
-
-  const dropdrownWidth = useCallback(() => {
-    if (containerRef.current) {
-      return `${containerRef.current.offsetWidth - 2}px`;
-    }
-    return 'auto';
-  }, []);
+  const handleChange = useCallback(
+    (option: OnChangeValue<Option, boolean>) => {
+      if (isMultiOption(option)) {
+        onChange(option.map((it) => it.value));
+      } else {
+        onChange([option!.value]);
+      }
+    },
+    [onChange],
+  );
 
-  const dropdrownPosition = useCallback(() => {
-    if (containerRef.current) {
-      return containerRef.current.offsetHeight + 1;
-    }
-    return 33;
-  }, []);
+  const Component = preventAdding ? Select : CreatableSelect;
+  const propsForCreatable = preventAdding
+    ? {}
+    : {
+        formatCreateLabel: CreateLabel,
+      };
 
   return (
-    <Label
-      as="span"
-      text={label}
-      width={width}
-      action={labelAction}
-      {...props}
-      onClick={() => !disabled && !isOpen && handleToggle()}
-    >
-      <EditableSelectContainer
-        ref={containerRef}
-        type={type}
-        label={label}
-        value={value}
-        search={search}
-        intent={intent}
-        isMulti={isMulti}
+    <FormControl isDisabled={disabled}>
+      {label && (
+        <FormLabel>
+          <Label
+            as="span"
+            text={label}
+            width={width}
+            action={labelAction}
+            sx={{
+              span: {
+                mb: 0,
+                mr: 0,
+              },
+            }}
+            {...props}
+          />
+        </FormLabel>
+      )}
+
+      <Component<Option, boolean>
+        variant={variant}
+        tagVariant="solid"
+        size="sm"
+        useBasicStyles
+        menuPortalTarget={document.querySelector('.chakra-portal') as any}
+        styles={{
+          menuPortal: (provided) => ({ ...provided, zIndex: 200 }),
+        }}
+        chakraStyles={{
+          menuList: R.mergeLeft({
+            my: 1,
+            py: 0,
+          }),
+          menu: R.mergeLeft({
+            my: 0,
+            py: 0,
+          }),
+          placeholder: R.mergeLeft({
+            fontSize: '12px',
+            color: 'gray',
+          }),
+          input: R.mergeLeft({
+            fontSize: '12px',
+          }),
+          option: R.mergeLeft({
+            fontSize: '12px',
+          }),
+          noOptionsMessage: R.mergeLeft({
+            fontSize: '12px',
+          }),
+          multiValue: R.mergeLeft({
+            bg: variant === 'white' ? 'grayShade3' : 'background',
+          }),
+        }}
+        closeMenuOnSelect={false}
+        openMenuOnFocus // needed for accessibility, e.g. trigger on a label click
+        isClearable={false} // removes clear button [X] that clears the whole select
```

Comment:
I think having this option to make single selects clearable would be good, for cases like filtering, ... to not having a separate 'None' option